src/pages/repositories.html

```diff
@@ -85,21 +85,21 @@ <h1>
 											tooltip-placement="top" tooltip-trigger="mouseenter"></i>
 									</span>
 									<span ng-hide="repository.type === 'system' || getDegradedReason()">
-										<a class="btn btn-link p-0"
+										<a class="btn btn-link p-0 edit-repository"
```

Comment:
If we follow the pattern, this should be `edit-repository-btn`

Reply:
Renamed